src/main/java/AreaCalculator.java

```diff
@@ -0,0 +1,3 @@
+public interface AreaCalculator {
+    public abstract double getArea();
```

Comment:
```suggestion
    double getArea();
```

све методы интерфейса без реализации по умолчанию абстрактные и публичные, намне нужно дополнительно это указывать